koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -717,6 +738,116 @@ LEFT JOIN postgres.{{schema | sqlsafe}}.reporting_awsaccountalias AS aa
 WHERE pds.ocp_source = {{ocp_source_uuid}} AND year = {{year}} AND month = {{month}}
 ;
 
+-- Put Node Network Costs into the Network unattributed namespace
+INSERT INTO hive.{{schema | sqlsafe}}.reporting_ocpawscostlineitem_project_daily_summary (
+    aws_uuid,
+    cluster_id,
+    cluster_alias,
+    data_source,
+    namespace,
+    node,
+    persistentvolumeclaim,
+    persistentvolume,
+    storageclass,
+    resource_id,
+    usage_start,
+    usage_end,
+    product_code,
+    product_family,
+    instance_type,
+    usage_account_id,
+    account_alias_id,
+    availability_zone,
+    region,
+    unit,
+    usage_amount,
+    data_transfer_direction,
+    currency_code,
+    unblended_cost,
+    markup_cost,
+    blended_cost,
+    markup_cost_blended,
+    savingsplan_effective_cost,
+    markup_cost_savingsplan,
+    calculated_amortized_cost,
+    markup_cost_amortized,
+    pod_cost,
+    project_markup_cost,
+    pod_labels,
+    tags,
+    aws_cost_category,
+    cost_category_id,
+    aws_source,
+    ocp_source,
+    year,
+    month,
+    day
+)
+SELECT
+    aws.uuid AS aws_uuid,
+    max(cluster_id),
+    max(cluster_alias),
+    max(data_source),
+    'Network unattributed' AS namespace,
+    max(node),
```

Comment:
Don't need the max here, this is grouped by node.
```suggestion
    node,
```